.gitignore

```diff
@@ -21,4 +21,8 @@ diff.py
 Vagrantfile
 *.sublime*
 coverage/
-.env
\ No newline at end of file
+.env
```

Comment:
I thought we were going to un-commit these changes and leave them as local until we were merging the relevant files in?